app/code/community/Algolia/Algoliasearch/Helper/Algoliahelper.php

```diff
@@ -109,52 +109,37 @@ public function mergeSettings($index_name, $settings)
         return $onlineSettings;
     }
 
-    public function handleTooBigRecords(&$objects, $index_name)
+    public function addObjects($objects, $indexName)
     {
-        $long_attributes = array('description', 'short_description', 'meta_description', 'content');
+        $currentCET = new DateTime('now', new DateTimeZone('Europe/Paris'));
+        $currentCET = $currentCET->format('Y-m-d H:i:s');
+        
+        $modifiedIds = array();
 
-        $good_size = true;
+        foreach ($objects as $key => $object) {
```

Comment:
Why?
The reason the loop is here I don't want to mix "add time" and "handle big records" features together in any other function. The only thing those "features" have in common is the "addObjects" method.